packages/rpc-core/src/rpc-methods/getRecentPerformanceSamples.ts

```diff
@@ -0,0 +1,26 @@
+import { Slot, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+type PerformanceSample = Readonly<{
+    /** Slot in which sample was taken at */
+    slot: Slot;
+    /** Number of transactions in sample */
+    numTransactions: U64UnsafeBeyond2Pow53Minus1;
+    /** Number of slots in sample */
+    numSlots: U64UnsafeBeyond2Pow53Minus1;
+    /** Number of seconds in a sample window */
+    samplePeriodSecs: number;
+    /** Number of non-vote transactions in sample. */
+    numNonVoteTransaction: U64UnsafeBeyond2Pow53Minus1;
+}>;
+
+type GetRecentPerformanceSamplesApiResponse = [PerformanceSample];
```

Comment:
I don't think this is right. This means ‘a one-element tuple that contains a `PerformanceSample`’.
```suggestion
type GetRecentPerformanceSamplesApiResponse = readonly PerformanceSample[];
```